index.html

```diff
@@ -287,6 +287,7 @@
     are defined in [[!DOM]].
   </p>
   <p>
+    <dfn data-cite="!ENCODING#concept-stream-push">push</dfn>,
```

Comment:
Note: you don't need "!" in front of citations. ReSpec knows when something is normative or informative ("artificial intelligence" lol). 

```suggestion
    <dfn data-cite="ENCODING#concept-stream-push">push</dfn>,
```

Reply:
Nice, I will improve the rest in a follow-up.